Makefile

```diff
@@ -70,12 +70,13 @@ build/obs:
 	osc checkout $(OBS_PROJECT)/$(OBS_PACKAGE) -o build/obs
 	rm -f build/obs/*.tar.gz
 	cp -rv packaging/obs/* build/obs/
-	sed -i 's/%%VERSION%%/$(VERSION)/' build/obs/_service
+	sed -i 's~%%VERSION%%~$(VERSION)~' build/obs/_service
```

Comment:
I think we a comment which state what we are doing with sed perhaps can be useful in future for maintaince

Reply:
Good catch! admittedly, it looks very obscure. I will add some comments about this.